src/templates/course.html

```diff
@@ -125,6 +140,55 @@
             }
         });
     }
+    function addTA() {
+        $.ajax({
+              url: '{{ url_for("add_TA") }}',
+              type: 'POST',
+              contentType: 'application/json',
+              data: JSON.stringify({
+                    'email': $("#TAEmail").val(),
+                    'courseID': parseInt("{{course.id}}"),
+                }),
+                success:  function(data) {
```

Comment:
Same comment. Better to return a failure from the python, then handle it with a "failure" function rather than "success"